### ISSUE TITLE
github_authtoken set but no updates/mentions on github

### ISSUE THREAD

**Issue by pape-io** [cider-ci/cider-ci#16]
github_authtoken is set but no updates/mentions/test-results appear on github ;(
v3.7

**Comment by DrTom** [comment]
Just checked. It works for us like a charm. 

Do you use a repository specific key or a general configured one? 

I could go into your machine and do some debugging. Would this be possible?

**Comment by eins78** [comment]
I also have issues with my repos (public, but under my username).
- It does not work with default token: https://github.com/eins78/json-roa-browser.git 
- It works where I made a specific per-repo key: https://github.com/eins78/json-roa-browser.git

**Comment by DrTom** [comment]
We can draw some deductions now: 

In principal 
 * per repo keys do work (Max) 
 * and general keys do work (Ci@zhdk).

I suspect there is something not right with the permissions on github. 

@eins78 : did you allow the corresponding user of the general key? 

@pape-io : It could be many things. We need to debug.

**Comment by eins78** [comment]
@DrTom Not sure what the 'corresponding user of the general key' would be (the bot account?), but I did not add anybody. I think the setup where it works was made according to the docs, using personal access token (at least I have one for cider in my account). 
Now that I looked it up, this personal token does not seem to have any 'scope', so it should also work for the other project (I'll try).

**Comment by pape-io** [comment]
- Per repo key does not work for our private repos (local)

<img width="1111" alt="bildschirmfoto 2015-10-20 um 08 39 51" src="https://cloud.githubusercontent.com/assets/851393/10599856/3a80fe6e-7706-11e5-96a1-3c2004efd417.png">

**Comment by DrTom** [comment]
@eins78  Yes, for ci.zhdk.ch it is the bot account https://github.com/ci-zhdk-ch.

Those keys always belong to an GitHub account. They don't live on their own. The corresponding user must be permitted to push to the repo. Other wise it can not update the stati.

**Comment by DrTom** [comment]
@pape-io 

1. Can you have a look at the files in `/var/local/cider-ci/server/notifier/log`? 

2. You are using SSH connections to GitHub, right?

**Comment by pape-io** [comment]
@DrTom 
```

[ERROR Oct-20-Tue_06:35:04 cider-ci.notifier.github-statuses clojure-agent-send-off-pool-41] ["java.net.URISyntaxException: Illegal character in scheme name at index 3: git@github.com:local-ch/location-app.git" ("cider_ci.notifier.github_statuses$amend_with_url_properties.invoke (github_statuses.clj:32)" "cider_ci.notifier.github_statuses$evaluate_update$fn__6138.invoke (github_statuses.clj:109)")]
[ERROR Oct-20-Tue_06:41:26 cider-ci.notifier.github-statuses clojure-agent-send-off-pool-42] ["java.net.URISyntaxException: Illegal character in scheme name at index 3: git@github.com:local-ch/location-app.git" ("cider_ci.notifier.github_statuses$amend_with_url_properties.invoke (github_statuses.clj:32)" "cider_ci.notifier.github_statuses$evaluate_update$fn__6138.invoke (github_statuses.clj:109)")]
```

**Comment by DrTom** [comment]
So we have the culprit. The service tries to infer the push location from you repository via dissecting it as an URI. That doesn't work. 

https://github.com/cider-ci/cider-ci_notifier/issues/1

**Comment by DrTom** [comment]
@pape-io  This should be fixed in 3.8.0-12.beta. Could you check it out and give us feedback?

**Comment by pape-io** [comment]
Can confirm: Notification bug fixed in 3.8.0-12.beta.
